### ISSUE TITLE
Length function is not working

### ISSUE THREAD

**Issue by AdrianMachado** [daveshanley/vacuum#417]
## Description

I tried creating a new rule that used `length`in my ruleset, as described [here](https://quobix.com/vacuum/functions/core/length/) 

## Problem

I copied and modified the `check-the-length` rule from the link above
```yaml

check-the-length:
    description: "Check the length of 'name' is at least 5 characters long"
    type: style
    recommended: true
    given: $.tags[*]
    then:
      field: name
      function: "length"
      functionOptions:
        min: 5
        max: 8
```
and ran it against Asana's OpenAPI file, but it did not work - I made sure there were violations in the OpenAPI file so I know the rule is likely just not running.

## Repro

Command
`vacuum spectral-report -r /Users/adrianmachado/Code/rate-my-openapi-cli/rulesets/rules.vacuum.yaml  example-specs/asana.json `

My ruleset is `extends: [[spectral:oas, all], [vacuum:owasp, all]]` with a few others added in, including that `check-the-length`.  I am pretty sure my other rules are working fine.

**Comment by daveshanley** [comment]
This is a bug, will fix.

**Comment by daveshanley** [comment]
This has been fixed in `v0.8.0` There were multiple issues at play.
